### PR TITLE
added Workflow system in introduction, both to the diagram and the prose

### PR DESCRIPTION
**Pull request by jandrieu**
***
<a href="https://pr-preview.s3.amazonaws.com/LegReq/vc-api/pull/353.html" title="Last updated on Feb 6, 2024, 8:42 PM UTC (1984045)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vc-api/353/f634b54...LegReq:1984045.html" title="Last updated on Feb 6, 2024, 8:42 PM UTC (1984045)">Diff</a>

### CODE REVIEW

File: diagrams/components.svg
Comment (TallTed) [inline diagrams/components.svg:1]: * There's an arrow labeled "receive verifiable credentials" that points from Holder Coordinator to Issuer Coordinator. I think this must point the other direction.
* If the "Any Authorized Party" hexagon moves up, and the 3 "Workflow service" parallelograms move into the inner field with the "Any Authorized Party" hexagon, then the "use exchange" and "create exchange" arrows can all be shifted slightly, such that none of them cross each other.
* If the "Holder" human moves left, and the "Storage Service (Holder)" cylinder moves into the triangle of "Holder" human, "Holder Coordinator" pentagon, and "Holder Admin" box, the "use wallet" and "configure" arrows between "Holder" human and "Holder Coordinator" pentagon and between "Holder" human and "Holder Admin" box can be adjusted somewhat to again make there be no crossings.

File: index.html [hunk 1]
```diff
@@ -464,6 +464,12 @@ <h3>Storage Services</h3>
             (Verifier) &bull; Storage Service (Holder)
           </strong>
         </p>
+      <section>
+        <h3>Workflow Service</h3>
+        <p>
+          The Workflow Service provides a way for coordinators to automate
+          specific interactions for specific users. Each role: Holder, Issuer, and Verifier, may run their own Workflow Service to create and manage exchanges that realize particular workflows. Administrators configure the workflow system to support particular flows. Then, when the business rules justify it, coordinators create exchanges at their Workflow Service and give authorized access to those exchanges to any party.
```
Comment (TallTed) [inline index.html:471]: Wrapped to match the rest of this source. Tweaked RFC2119 language, as this section is informative. Other changes are not properly highlighted, but do exist.
```suggestion
The Workflow Service provides a way for coordinators to automate specific
interactions for specific users. Each role (Holder, Issuer, and Verifier)
can run their own Workflow Service to create and manage exchanges that
realize particular workflows. Administrators configure the workflow system
to support particular flows. Then, when the business rules justify it,
coordinators create exchanges at their Workflow Service and give authorized
access to those exchanges to any party. 
```

**Review by msporny — APPROVED** [review]
Approving w/ @TallTed's changes applied.

The diagram is getting to the point that we're going to have to refactor it. It gives the impression that the ecosystem is more complex than it is due to the crisscrossing lines.

**Comment by TallTed** [comment]
> The diagram is getting to the point that we're going to have to refactor it. It gives the impression that the ecosystem is more complex than it is due to the crisscrossing lines.

Yes. Note that I've provided [guidance for minor movements of existing elements](https://github.com/w3c-ccg/vc-api/pull/353#discussion_r1425929555) that will eliminate all line crossings.

**Review by dlongley — APPROVED** [review]
+1 w/ @TallTed's suggestions.

File: diagrams/components.svg
Comment (jandrieu) [reply diagrams/components.svg:1]: Agreed. Incoming graphic soon.

**Comment by jandrieu** [comment]
@TallTed Fixes in. 

Let me know if I got all of your suggestions on the diagram.

**Review by TallTed — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**